### PR TITLE
[niina] Use simulate in pre_hyouka()

### PR DESCRIPTION
**Pull request by peria**
#125

### CODE REVIEW

**Comment by peria** [comment]
PTL

File: src/cpu/test_lockit/coma.cc [hunk 1]
```diff
@@ -1283,62 +1278,6 @@ int COMAI_HI::pre_hyouka(const TLColor ba3[6][kHeight], TLColor tsumo[], int zen
...
-                                continue;
-                            chain = 0;
-                            chousei_syoukyo_2(ba, setti_basyo, &chain, dabuchk, &ichiren_kesi, &score);
-                        }
-                    } // cyy
-                } // cc
-            } // aa
-        }
-
-    } // m_aite_hakka_honsen==0
```
Comment (mayah) [inline src/cpu/test_lockit/coma.cc]: Why these are removed? They don't have any meaning, maybe?

**Comment by mayah** [comment]
Could you rebase on ToT?

File: src/cpu/test_lockit/coma.cc (hunk 1 above)
Comment (peria) [inline src/cpu/test_lockit/coma.cc]: This part tries to put 2 puyos and simulate rensa, but its result is not used.

**Comment by peria** [comment]
rebased

**Comment by mayah** [comment]
LGTM.

**Pull request merged**